qa/workunits/ceph-helpers.sh

```diff
@@ -1282,6 +1302,70 @@ function test_display_logs() {
 }
 
 #######################################################################
+##
+# Spawn a command in background and save the pid in the variable name
+# passed in argument
+#
+# Example:
+#   pids1=""
+#   run_in_background pids1 bash -c 'sleep 1; exit 1'
+#
+# @param pid_variable the variable name (not value) where the pids will be stored
+# @param $@ the command to execute
+# @return only the pid_variable output should be considered and used with **wait_background**
+#
+function run_in_background() {
+    local pid_variable=$1
+    shift;
+    "$@" & eval "$pid_variable+=\" $!\""
+}
```

Comment:
nit: could you group run_in_background and test_run_in_background together and separate them from wait_background and test_wait_in_background with a ####################################################################### line ? So it is like all other functions in the file. Exactly one function, one test and the documentation :-)